dkist/net/attr_walker.py

```diff
@@ -143,6 +147,20 @@ def _(wlk, attr, params):
     return params.update({'embargoEndDateMin': attr.start.isot,
                           'embargoEndDateMax': attr.end.isot})
 
+@walker.add_applier(SpectralSampling)
+def _(wlk, attr, params):
+    return params.update({'averageDatasetSpectralSamplingMin': attr.min,
+                          'averageDatasetSpectralSamplingMax': attr.max})
```

Comment:
These too will be quantities you will need to cast them back to float with `to_value`.